internal/apptracker/app_tracker.go

```diff
@@ -0,0 +1,6 @@
+package apptracker
+
+type AppTracker interface {
+	CaptureMessage(message string)
+	CaptureException(exception error)
```

Comment:
Can we also add the context here?

```suggestion
	CaptureMessage(ctx context.Context, message string)
	CaptureException(ctx context.Context, exception error)
```

Reply:
Done